src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -130,6 +131,10 @@ export default Backbone.Model.extend({
 
     if (this.id === 'shared-by' || !clinicianId) {
       filters[this.id].role = roleId;
+
+      if (noOwner) {
+        filters[this.id].clinician = '00000000-0000-0000-0000-000000000000';
```

Comment:
I just ran across this https://www.npmjs.com/package/uuid#uuidnil. We should use it